searchableselect/views.py

```diff
@@ -1,13 +1,40 @@
 try:
     # Django <=1.9
     from django.db.models.loading import get_model
+
+    # Django 1.6.*
+    import django
+    if django.VERSION[0] == 1 and django.VERSION == 6:
```

Comment:
Did you wanted to write `django.VERSION[0] == 1 and django.VERSION[1] == 6` (missed `[1]`) ?

Reply:
Can I add new commit to this PR?